### PR TITLE
Add Monitor Link shares script to the not-supported folder

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by szaimen** [comment]
This is ready 👍

**Comment by szaimen** [comment]
Now it is ready 👍

File: network/geoblock.sh [hunk 1]
```diff
@@ -32,10 +32,6 @@ else
     # Removal
     find /var/scripts -type f -regex \
 "$SCRIPTS/202[0-9]-[01][0-9]-Maxmind-Country-IPv[46]\.dat" -delete
-    if is_this_installed jq
-    then
-        apt purge jq -y
```
Comment (enoch85) [inline network/geoblock.sh:37]: Why not, `install_if_not()`?

Comment (szaimen) [reply network/geoblock.sh:37]: Maybe in a follow up but I actually removed those lines because jq is no longer part of the script.

**Pull request merged**